content/en/device-provisioning-service/client-library.md

```diff
@@ -27,19 +27,18 @@ Please examine the contents of the provided pkg-config (`.pc`) file and install
 
 The API is defined in the public header files provided in the distributed package.
 
-`dps.h`:
+[dps.h](../../dps/dps.h):
```

Comment:
@ondrejtomcik I think there should be a way of rendering the file, so Copy&Paste is not needed, but I'm not familiar with the rendering mechanism we use here. Do you know if it is possible?